COCO_DATA_FORMAT.md

```diff
@@ -94,3 +94,18 @@ Specifically, **only** image files are supported for the label files. The ground
     ]
 }
 ```
+
```

Comment:
The code in this repo require update as well to be able to consume this new format of data

Reply:
along with the readme file

Reply:
Made the corresponding changes, thanks!